ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -107,6 +107,25 @@ spec:
         - name: pyxis-ssl-credentials
           workspace: pyxis-ssl-credentials
 
+    # Update cert project related data
+    - name: update-cert-project-data
```

Comment:
Just a nitpick: WDYT about naming it "update-cert-project-status"? It would be more accurate.